src/main/java/duke/DialogBox.java

```diff
@@ -1,47 +1,60 @@
 package duke;
 
+import java.io.IOException;
+import java.util.Collections;
+
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
 import javafx.geometry.Pos;
 import javafx.scene.Node;
 import javafx.scene.control.Label;
-import javafx.scene.layout.HBox;
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
```

Comment:
While it is great that you grouped the respective import statements, appropriate spacing could make it easier to browse the list and determine the dependencies, especially since there are many imports.